fx.go

```diff
@@ -11,9 +15,12 @@ import (
 	"github.com/urfave/cli"
 )
 
+const Version = "0.7.3"
+
 var cfg *config.Config
 
 func init() {
+	go checkForUpdate()
```

Comment:
Since the checking process is running in a `goroutine`, so only **fx** running command takes more time than `checkForUpdate()`, otherwise the user has no chance to see the result of `checkForUpdate()`, right?

Reply:
Right, this feature is designed as a side effect to let user know the update **eventually**.

Or do you have any idea how to notify user whenever a update is available?

Reply:
I have no better solution either, let's adopt this solution, thanks for your great contribution here.